docs/sec-blueprint/08-Hardening.md

```diff
@@ -580,24 +1014,228 @@ Kernel Hardening
+|  Who and whoami   |  /bin/whoami                              |  INCLUDE               |  EXCLUDE                     |
+|  ps               |  /bin/ps                                  |  INCLUDE               |  EXCLUDE                     |
+|  lsmod            |  /sbin/lsmod                              |  INCLUDE               |  EXCLUDE                     |
+|  install          |  /bin/install                             |  INCLUDE               |  EXCLUDE                     |
+|  logger           |  /bin/logger                              |  INCLUDE               |  EXCLUDE                     |
+|  ps               |  /bin/ps                                  |  INCLUDE               |  EXCLUDE                     |
+|  rpm              |  /bin/rpm                                 |  INCLUDE               |  EXCLUDE                     |
+|  Iostat           |  /bin/iostat                              |  INCLUDE               |  EXCLUDE                     |
+|  find             |  /bin/find                                |  INCLUDE               |  EXCLUDE                     |
+|  Chgrp            |  /bin/chgrp                               |  INCLUDE               |  EXCLUDE                     |
+|  Chmod            |  /bin/chmod                               |  INCLUDE               |  EXCLUDE                     |
+|  Chown            |  /bin/chown                               |  INCLUDE               |  EXCLUDE                     |
+|  killall          |  /bin/killall                             |  INCLUDE               |  EXCLUDE                     |
+|  top              |  /bin/top                                 |  INCLUDE               |  EXCLUDE                     |
+|  stbhotplug       |  /sbin/stbhotplug                         |  INCLUDE               |  EXCLUDE                     |
+
+Note: The following Unix/Linux utilities shall be permitted as they are
+often used in the start-up scripts and for USB logging. If any of these
+utilities are not required by the device then those should be removed.
+
+  **sed, awk, cut, df, dmesg, echo, fdisk, grep, mkdir, mount (vfat),
+  printf, tail, tee, test (directory), test (file)**
+
+Root Access 
+------------
+
+The main applications, those that provide the principal functionality of
+the embedded device, **should not execute** with root privileges.
+
+If the main application are allowed to execute at elevated privileges,
+then the entire system is at the mercy of the said application’s good
+behaviour. Problems arise when an application is compromised and able to
+execute commands which could consistently and persistently compromise
+the system by implanting rogue applications.
+
+It is suggested that the middleware and the UI should run in a context
+on a user with lowered privileges and all persistent resources should be
+maintained with such privileges.
+
```

Comment:
by "... with such privileges" should I understand " ... without any capability" ?